### DISCUSSION TITLE
Allow bypassing the upload window

### DISCUSSION THREAD

**Discussion by thiswillbeyourgithub** [astubenbord/paperless-mobile#412] [Feature Requests]
My workflow so far is to take pictures of each pages in a single document, then click upload, then simply confirm because I do the whole tagging etc directly on the computer. But when I have lots of documents, it gets annoying.

And for that use case, I would prefer the "upload" button to upload directly without asking me for tags etc.

I think that would be worth adding a new toggle switch in the settings for that.

**Comment by astubenbord** [comment]
There already is a setting for files shared with the app, do you think it makes sense to also make it control uploading scans taken within the app?
